src/style.css

```diff
@@ -1 +1,101 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 400;
+  font-style: normal;
+}
+
+.body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It is usually a good practice to provide a list of similar fonts that the browser can fall back to in case the primary font fails to load. This ensures that your website will still look good, even if the font files are not available for some reason.